docs/source/upgrade-guide/index.md

```diff
@@ -218,6 +218,14 @@ The `.stories.mdx` extension is no longer supported.
 Although it is technically possible to keep the old version running, the script `volto-update-deps` will try to update to Storybook 8 every time you run it.
 ```
 
+### Form component passes down the id of the current fieldset
+
+There was a bug that was provoking to have `id` attributes with spaces on it, if the a fieldset was longer than one word.
```

Comment:
It's not clear to me what the symptom was, so how can I identify it?

```suggestion
A bug where a fieldset's `title` contains spaces would cause [INSERT DESCRIPTION OF SYMPTOM HERE] has been fixed.
```

Reply:
@stevepiercy in order to fix this one, I ended up by retouching the whole thing again.
Please take a look again.